tests/NuGet.IndexingTests/TestSupport/TokenAttributes.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.IndexingTests.TestSupport
+{
+    public class TokenAttributes
+    {
+        public TokenAttributes()
+        {
+        }
+
+        public TokenAttributes(string term, int startOffset, int endOffset) : this(term, startOffset, endOffset, null)
```

Comment:
this constructor is unnecesssary, for a test project feel free to use default parameters